spring-vault-core/src/main/java/org/springframework/vault/support/CertificateBundle.java

```diff
@@ -46,14 +46,43 @@
 
 	private final String privateKey;
 
+	private final String privateKeyType;
+
 	private final List<String> caChain;
 
+	/**
+	 * Create a new {@link CertificateBundle}.
+	 * @param serialNumber the serial number.
+	 * @param certificate the certificate.
+	 * @param issuingCaCertificate the issuing CA certificate.
+	 * @param caChain the CA chain.
+	 * @param privateKey the private key.
+	 * @deprecated since 2.3.3, use {@link #CertificateBundle(String, String, String, List, String, String)} instead.
+	 */
+	@Deprecated
+	CertificateBundle(String serialNumber, String certificate, String issuingCaCertificate,
```

Comment:
We do not need to reintroduce a deprecated constructor variant as the original constructor was package-private so we're free to change the constructors.